mava/configs/system/q_learning/rec_iql.yaml

```
@@ -0,0 +1,32 @@
+# --- Defaults REC-IDQN ---
+total_timesteps: 20_000_000
+num_updates: 1000 # total_timesteps takes preference
+seed: 1
+
+# --- Agent observations ---
+add_agent_id: True
+
+# --- RL hyperparameters ---
+buffer_min_size: 132  # Number of steps to take with random actions at the start of training.
+update_batch_size: 1 # Number of vectorised gradient updates per device.
+
+rollout_length: 2 # Number of environment steps per vectorised environment.
+epochs: 2 # Number of learn epochs per training data batch.
+
+# sizes
+buffer_size: 5_000  # size of the replay buffer. Note: total size is this * num_devices
+sample_batch_size: 32 #as in jaxmarl paper
+
+# learning rates
+q_lr: 3e-4  # the learning rate of the Q network network optimizer
+# other
+hard_update: True
+update_period: 200
+tau: 0.01  # smoothing coefficient for target networks
```

I suspect we have this for if we want to do soft target updates, so I am just setting it to be empty for now. Please ignore if this is not the case. 
```suggestion
tau: ~  # smoothing coefficient for target networks. used for soft target network updates
```

---

It is for soft updates, but I do want the option to switch. Will therefore be leaving it there for now.